### PR TITLE
Add Node-RED section in README

### PR DESCRIPTION
**Pull request by mattieha**
As suggested by @leikoilja adds a Node-RED section to the readme .
Link to a blog post about a sublfow I created to get the alarms and timers as notifications on my phone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,6 +239,12 @@ Make sure that you have your Home Assistant Container network set to `host`, as
 
 - [Google Home timers card](https://github.com/DurgNomis-drol/google_home_timers_card) by [@DurgNomis-drol](https://github.com/DurgNomis-drol)
 
+## Node-RED Flows
```
Comment (KapJI) [inline README.md:242]: Add it to table of contents 🙂

**Comment by DurgNomis-drol** [comment]
This is awesome! Thanks for sharing this with everyone 🚀

File: README.md (hunk 1 above)
Comment (mattieha) [reply README.md:243]: Nice catch, it's added now.

**Review by KapJI — APPROVED** [review]
Awesome, thanks! 🎉

**Pull request merged**